lein-template/src/duct/duct_template.clj

```diff
@@ -9,42 +9,47 @@
    "src/{{dirs}}/handler"
    "test/{{dirs}}/handler"])
 
-(defn example-profile [{:keys [profiles]}]
-  {:vars {:example? true}
+(defn example-profile [{:keys [profiles project-ns]}]
+  {:vars {:example? true
+          :cascading-routes [(keyword (str project-ns ".handler/example"))]}
    :templates
    (cond
      (profiles :site)
-     {"src/{{dirs}}/handler/example.clj"       (resource "example/handler.clj")
+     {"src/{{dirs}}/handler/example.clj" (resource "example/handler.clj")
```

Comment:
You have a bunch of formatting changes where you remove aligned forms. Can you ensure that forms are aligned correctly?